src/stripe.worker.ts

```diff
@@ -1,8 +1,16 @@
 import Stripe = require('./stripe.common');
+import StripeEmitter = require('./StripeEmitter');
 
 Stripe.createHttpClient = Stripe.createFetchHttpClient;
 Stripe.webhooks._createCryptoProvider = Stripe.createSubtleCryptoProvider;
 
+// StripeEmitter uses the Event Web API. `Event` is not available
+// in the global scope until Node 15, so we set `createEmitter` here
+// to avoid reference errors.
+Stripe._platformFunctions.createEmitter = (): StripeEmitter => {
+  return new StripeEmitter(new EventTarget());
```

Comment:
What fails if we move this logic to DefaultPlatformFunctions?

Reply:
We might need a base type for platform functions and then 2 implementations: one for node and one for workers. In that way the node implementation won't have to inherit all dependencies of the worker implementation.

Reply:
> What fails if we move this logic to DefaultPlatformFunctions?
This used to fail because it imports `StripeEmitter`, where we extend `Event` for `_StripeEvent`. `Event` is not available globally until Node 15, so this failed for Node 12 and 14 tests.

I followed your suggestion and created a `PlatformFunctions` base type that has `NodePlatformFunctions` and `WebPlatformFunctions` implementations in https://github.com/stripe/stripe-node/pull/1668/commits/fafb866900d20f3001c12537d12305df140e8f20